Gemfile.lock

```diff
@@ -3,24 +3,25 @@ GEM
   specs:
     CFPropertyList (3.0.5)
       rexml
-    actionpack (7.0.4)
-      actionview (= 7.0.4)
-      activesupport (= 7.0.4)
-      rack (~> 2.0, >= 2.2.0)
+    actionpack (6.1.7.2)
```

Comment:
Why are we downgrading things?

Reply:
After some investigation I still have no idea why bundle install/update downgrade some of gems. will investigate